tests/test_optim/test_scheduler/test_param_scheduler.py

```diff
@@ -438,6 +438,24 @@ def test_cosine_restart_scheduler(self):
             eta_min=0)
         self._test_scheduler_value(scheduler, targets, epochs=10)
 
+        epochs = 10
+        t = 10
+        eta_min = 1e-7
+        single_targets = [
+            eta_min + (0.05 - eta_min) * (1 + math.cos(math.pi * x / t)) / 2
+            for x in range(epochs)
+        ]
+        targets = [
+            single_targets, [x * self.layer2_mult for x in single_targets]
+        ]
+        scheduler = CosineRestartParamScheduler(
+            self.optimizer,
+            param_name='lr',
+            periods=[t],
```

Comment:
The last epoch will not be tested in `_test_scheduler_value`, therefore `0` learning rate will not be compared with `1e-7` at the tenth epoch.